### PR TITLE
replace kernel.root_dir with kernel.project_dir

### PR DESCRIPTION
**Pull request by xabbuh**
see #7768

### CODE REVIEW

File: reference/configuration/doctrine.rst [hunk 1]
```diff
@@ -398,7 +398,7 @@ The following block shows all possible configuration keys:
                 # the DBAL driverOptions option
                 options:
                     foo: bar
-                path:                 '%kernel.root_dir%/data/data.sqlite'
+                path:                 '%kernel.project_dir%ta/data.sqlite'
```
Comment (iltar) [inline reference/configuration/doctrine.rst]: `%kernel.project_dir%ta/` should be `%kernel.project_dir%/`

**Review by javiereguiluz — APPROVED** [review]
:+1:

File: reference/configuration/doctrine.rst (hunk 1 above)
Comment (javiereguiluz) [inline reference/configuration/doctrine.rst]: The original was `'%kernel.root_dir%/data/data.sqlite'`, so the new one should be `'%kernel.project_dir%/app/data/data.sqlite'`

**Comment by javiereguiluz** [comment]
Thanks Christian.

**Pull request merged**